File: api/test_connection.py
import psycopg2

try:
    conn = psycopg2.connect(
        host="localhost",
        database="postgres",
        user="Senanu",
        password="Senanu12",
        port="5432"
    )
    print("✅ Database connection SUCCESSFUL!")
    conn.close()
except Exception as e:
    print(f"❌ Connection failed: {e}")

File: api/test-connection.js
const { Pool } = require('pg');
require('dotenv').config();

const pool = new Pool({
  user: process.env.DB_USER,
  host: process.env.DB_HOST,
  database: process.env.DB_NAME,
  password: process.env.DB_PASSWORD,
  port: process.env.DB_PORT,
});

console.log('Trying to connect with:');
console.log('User:', process.env.DB_USER);
console.log('Database:', process.env.DB_NAME);

pool.query('SELECT NOW()')
  .then(res => {
    console.log('Connection successful!', res.rows[0]);
    pool.end();
  })
  .catch(err => {
    console.error('Connection failed:', err.message);
    pool.end();
  });
